public/angular/templates/sources/modal.html

```diff
@@ -39,24 +39,35 @@ <h4 class="modal-title">
             </div>
           </div>
 
-          <div class="form-group" ng-show="source.media == 'smsgh' || source.media == 'twitter' || source.media == 'whatsapp' ">
+          <!-- <div class="form-group" ng-show="source.media && source.media != 'twitter' && source.media == 'crowdtangle'">
+            <label for="apikey" class="col-lg-2 control-label" translate>API Key</label>
+            <div class="col-lg-8">
+                <input type="text" ng-model="source.apikey" class="form-control" id="apikey" name="apikey" placeholder="{{ 'Enter dashboard key' | translate }}" ng-required="source.media == 'crowdtangle'">
+              <div class="help-block" translate>The Dashboard API Key for this source.</div>
+              <div class="has-error" ng-show="showErrors() && form.apikey.$invalid">
+                <div class="help-block" ng-show="form.apikey.$error.required" translate>Dashboard API Key is required.</div>
+              </div>
+            </div>
+          </div> -->
+
+          <div class="form-group" ng-show="source.media == 'smsgh' || source.media == 'twitter' || source.media == 'crowdtangle' || source.media == 'whatsapp' ">
             <label for="keywords" class="col-lg-2 control-label" translate>Keywords</label>
             <div class="col-lg-8">
               <textarea ng-model="source.keywords" class="form-control" id="keywords" name="keywords" placeholder="{{ 'Enter keywords' | translate }}" ng-required="source.media == 'twitter'">
               </textarea>
               <div ng-show="source.media == 'twitter'" class="help-block" translate="sources_modal.help"></div>
               <div class="has-error" ng-show="showErrors() && form.keywords.$invalid">
-                <div class="help-block" ng-show="form.keywords.$error.required">Keywords are required for twitter sources.</div>
+                <div class="help-block" ng-show="form.keywords.$error.required">Keywords are required for Twitter sources.</div>
               </div>
             </div>
           </div>
 
-          <div class="form-group" ng-show="source.media && source.media != 'twitter' && source.media != 'smsgh' && source.media != 'whatsapp'">
+          <!-- <div class="form-group" ng-show="source.media && source.media != 'twitter' && source.media != 'crowdtangle' && source.media != 'smsgh' && source.media != 'whatsapp'">
```

Comment:
Why is this commented out? Can it be removed?

Reply:
Removed them.